src/server/index.jsx

```diff
@@ -61,6 +65,36 @@ server
  */
 
 if (process.env.APP_ENV === 'local') {
+  const errorWhenAccessingFile = async dataFilePath => {
```

Comment:
As nice as this is, as this is a dev route I dont think we need to be this safe or include a new dependancy. The following should be fine IMO

```
res.sendFile(path, {}, error => {
  if (error) {
    logger.error(error
    res.status(500).send('Unable to find file.');
  }
});
``